src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
**You did great implementation on this project.** 

- [ ] However, based on the requirement we need to remove all boilerplate codes including [styles](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=App%20(text%2C%20images%2C-,styles,-).). Kindly remove default styling from app css and index css :)

Reply:
@Rustamxon7  Hi! Just I need to remove this line "import  './App.css' " from the App.js file. Do I need to do anything else?